starboard/android/apk/app/src/main/java/dev/cobalt/coat/CobaltHttpHelper.java

```diff
@@ -25,6 +25,7 @@
 
 /** Helper class that implements an HTTP POST function used by DRM one-time provisioning. */
 public class CobaltHttpHelper {
+  private static final int HTTP_TIMEOUT_MILLIS = 500;
```

Comment:
I'd probably give it more grace time here. https://screenshot.googleplex.com/6TNaRDnoVig5XtY